crates/ty_python_semantic/src/semantic_index/expression.rs

```diff
@@ -62,6 +62,9 @@ pub(crate) struct Expression<'db> {
     count: countme::Count<Expression<'static>>,
 }
 
+// The Salsa heap is tracked separately.
+impl get_size2::GetSize for Expression<'_> {}
```

Comment:
Same as for `File`. Can't we derive `GetSize` here because it would be applied on the `Expression(salsa::Id)` type?